Hello.java

```diff
@@ -2,6 +2,6 @@
 {
    public static void main(String args[])
```

Comment:
build a new branch to finish this assignment, give the branch a meaningful name, use "#" to refer to issue link